tin/templates/assignments/show.html

```diff
@@ -40,14 +40,16 @@ <h2 class="left">{% if assignment.is_quiz %}[QUIZ] {% endif %}{{ assignment.name
         <a class="right tin-btn" href="{% url 'assignments:download_log' assignment.id %}">Download log</a>
       {% endif %}
     {% endif %}
-    {% if assignment.is_quiz %}
-      {% if quiz_accessible or is_teacher or request.user.is_superuser %}
-        <a class="right tin-btn"
-           onclick="return confirm('You are about to begin an assessment. All activity on your device will be monitored. If you open any other tabs or attempt to use any other applications, your quiz might be stopped and your teacher will be notified.')"
-           href="{% url 'assignments:quiz' assignment.id %}">Begin quiz</a>
+    {% if "w" in course.permission or is_teacher or request.user.is_superuser %}
```

Comment:
This line is wrong - it never checks whether the course is actually archived, banning submissions to all courses (unless they're specifically read-write). Backend logic seems fine.

Reply:
Nice catch, didn't notice that!